extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -156,12 +156,12 @@ class AmpStickyAd extends AMP.BaseElement {
    * @private
    */
   onScroll_() {
-    if (isExperimentOn(this.win, EARLY_LOAD_EXPERIMENT)) {
+    const scrollTop = this.viewport_.getScrollTop();
+    if (isExperimentOn(this.win, EARLY_LOAD_EXPERIMENT) && scrollTop > 1) {
```

Comment:
The problem is that iOS often offsets to 1 to avoid scroll freeze issue. See https://github.com/ampproject/amphtml/blob/master/src/service/viewport-impl.js#L1478